pkg/config/config.go

```diff
@@ -76,57 +105,128 @@ func SaveToFile(config *Config, filename string) error {
-				log.Fatalf("Validation failed: invalid secondary email (user: %s)", user.PrimaryEmail)
+				//return fmt.Errorf("invalid format of secondary email (user: %s)", user.PrimaryEmail)
+				allTheErrors = append(allTheErrors, fmt.Errorf("secondary email is not a valid email-address (user: %s)", user.PrimaryEmail))
+			}
+		}
+
+		if user.RecoveryEmail != "" {
+			if re.MatchString(user.RecoveryEmail) == false {
+				//return fmt.Errorf("invalid format of recovery email (user: %s)", user.PrimaryEmail)
+				allTheErrors = append(allTheErrors, fmt.Errorf("recovery email is not a valid email-address (user: %s)", user.PrimaryEmail))
+			}
+		}
+
+		if len(user.Aliases) > 0 {
+			for _, alias := range user.Aliases {
+				if re.MatchString(alias) == false {
+					//return fmt.Errorf("invalid format of alias email (user: %s)", user.PrimaryEmail)
+					allTheErrors = append(allTheErrors, fmt.Errorf("alias email is not a valid email-address (user: %s)", user.PrimaryEmail))
+				}
+			}
+		}
+
+		if user.Employee.ManagerEmail != "" {
+			if re.MatchString(user.Employee.ManagerEmail) == false {
+				//return fmt.Errorf("invalid format of the manager's email (user: %s)", user.PrimaryEmail)
+				allTheErrors = append(allTheErrors, fmt.Errorf("manager's email is not a valid email-address (user: %s)", user.PrimaryEmail))
+			}
+		}
+
+		if user.RecoveryPhone != "" {
+			if re164.MatchString(user.RecoveryPhone) == false {
+				//return fmt.Errorf("invalid format of recovery phone (user: %s). The phone number must be in the E.164 format, starting with the plus sign (+). Example: +16506661212.", user.PrimaryEmail)
+				allTheErrors = append(allTheErrors, fmt.Errorf("invalid format of recovery phone (user: %s). The phone number must be in the E.164 format, starting with the plus sign (+). Example: +16506661212.", user.PrimaryEmail))
 			}
 		}
+
 		userEmails = append(userEmails, user.PrimaryEmail)
 	}
 
-	// TODO: validate orgunits & groups
 	// validate groups
 	groupEmails := []string{}
 	for _, group := range c.Groups {
 		if util.StringSliceContains(groupEmails, group.Email) {
-			log.Fatalf("Validation failed: duplicate group email defined (%s)", group.Email)
+			//return fmt.Errorf("duplicate group email defined (%s)", group.Email)
+			allTheErrors = append(allTheErrors, fmt.Errorf("duplicate group email defined (%s)", group.Email))
 		}
 
 		if re.MatchString(group.Email) == false {
-			log.Fatalf("Validation failed: invalid group email (%s)", group.Email)
+			//return fmt.Errorf("invalid group email (%s)", group.Email)
+			allTheErrors = append(allTheErrors, fmt.Errorf("group email is not a valid email-address (%s)", group.Email))
+		}
+
+		if !(strings.EqualFold(group.WhoCanContactOwner, "ALL_IN_DOMAIN_CAN_CONTACT") || strings.EqualFold(group.WhoCanContactOwner, "ALL_MANAGERS_CAN_CONTACT") || strings.EqualFold(group.WhoCanContactOwner, "ALL_MEMBERS_CAN_CONTACT") || strings.EqualFold(group.WhoCanContactOwner, "ANYONE_CAN_CONTACT")) {
```

Comment:
Just a question: The validation is case-insensitive, so does GSuite also not care or do we later force the user-configured value to be uppercase?

Reply:
Indeed, it is case sensitive. My mistake. Changed to require upper-case values. 
I hope this fixes it: https://github.com/kubermatic-labs/gman/pull/11/commits/4b47f836cb50d908839f5ffef246927836676bce#diff-091ace02b49a46f55f2586cfec89a160R195